script.js

```diff
@@ -155,3 +155,26 @@ for (let i = 0; i < projectBtns.length; i += 1) {
   const btn = projectBtns[i];
   btn.addEventListener('click', () => openPopUp(i));
 }
+
+// FORM VALIDATION LOGIC
+
+const form = document.getElementById('contact-form');
+
+function showError() {
+  const error = document.querySelector('small');
+  error.className = 'show';
+  error.innerText = 'Please, use only lowercase';
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const emailForm = form.elements['user-email'];
+  let emailValid = false;
+  if (emailForm.value === emailForm.value.toLowerCase()) {
+    emailValid = true;
+  }
+  if (emailValid) {
+    form.submit();
+  }
+  showError();
+});
```

Comment:
- [ ] When using an email with uppercase letters the form is submitted normally and no error message is displayed, kindly fix that issue.

Reply:
Hello @algerina! 
Can you explain it a bit more? I just tested it and the error message does show
<img width="435" alt="image" src="https://github.com/Estete9/Fullstack-Portfolio/assets/23634070/beb5856c-1779-4143-b224-fb1384c05905">